pkg/exposer/csi_snapshot.go

```diff
@@ -455,3 +485,42 @@ func (e *csiSnapshotExposer) createBackupPod(ctx context.Context, ownerObject co
 
 	return e.kubeClient.CoreV1().Pods(ownerObject.Namespace).Create(ctx, pod, metav1.CreateOptions{})
 }
+
+func toSystemAffinity(loadAffinity *nodeagent.LoadAffinity) *corev1.Affinity {
```

Comment:
Why we cannot use the `corev1.Affinity` directly rather than defining our own type?

Reply:
I think there are two questions:

- Why do we wrap `metav1.LabelSelector`/`corev1.Affinity` instead of using it directly? -- We may need to add some other controllers that `metav1.LabelSelector`/`corev1.Affinity` cannot provide in future, so we add a simple wrap here. Whenever we want to change, we just append new fields so as to avoid break changes.
- Why don't we use `corev1.Affinity` directly? -- `corev1.Affinity` is more complex than we want to provide to users, e.g., we only use `RequiredDuringSchedulingIgnoredDuringExecution`, so we need to limit users input